diagnostic.md

```diff
@@ -57,6 +73,7 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
+$('li').parent().parent().hide('li');
```

Comment:
Very close! You don't need the `li` at the end. So, `$('li').parent().parent().hide();`